cpu/k60/include/system_MK60DZ10.h

```diff
@@ -1 +0,0 @@
-system_MK60D10.h
```

Comment:
On RIOT master branch system_MK60DZ10.h is a link, in my fork it is a file exactly equal to the linked file, so I think this generated the problem.
It is not clear to me if it is related to some wrong setting of mine or it is something else, very strange.